ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -886,10 +886,17 @@ bool UncoupledProcessesTimeLoop::solveCoupledEquationSystemsByStaggeredScheme(
     bool coupling_iteration_converged = true;
     for (unsigned global_coupling_iteration = 0;
          global_coupling_iteration < _global_coupling_max_iterations;
-         global_coupling_iteration++)
+         global_coupling_iteration++, _global_coupling_conv_crit->reset())
     {
+        if (global_coupling_iteration == 0)
```

Comment:
Changed accordingly by amending the commit of `` [CPL] Improved the convergence ...``,  aada847.